src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/PersonAggregate/SavedFilterTests.cs

```diff
@@ -29,21 +29,13 @@ public void Setup()
         public void Constructor_SetsProperties()
         {
             Assert.AreEqual(TestPlant, _dut.Plant);
-            Assert.AreEqual(ProjectId, _dut.ProjectId);
+            Assert.AreEqual(ProjectId, _dut.ProjectId.Value);
             Assert.AreEqual(Title, _dut.Title);
             Assert.AreEqual(Criteria, _dut.Criteria);
             Assert.AreEqual(DefaultFilterValue, _dut.DefaultFilter);
         }
 
 
```

Comment:
remove extra blank lines

Reply:
Didn't see any - but file changed anyway due to new tests